cmd/token_test.go

```diff
@@ -72,14 +80,78 @@ func TestTokenWithGCEAK(t *testing.T) {
 			}
 			defer mockAttestationServer.Stop()
 
-			RootCmd.SetArgs([]string{"token", "--algo", op.algo, "--output", secretFile1, "--asAddr", mockAttestationServer.server.URL})
+			//redirect cloud log from http request to stdout
+			old := os.Stdout
+			r, w, _ := os.Pipe()
+			os.Stdout = w
```

Comment:
We should be able to avoid the pipe redirection and sync primitives by using the output flag properly in a test directory https://pkg.go.dev/testing#T.TempDir